syntaxes/odoc.json

```
@@ -0,0 +1,171 @@
+        },
+        {
+          "comment": "emphasize text",
+          "begin": "\\{e\\b",
+          "end": "\\}",
+          "contentName": "emphasis",
+          "patterns": [{ "include": "#markup" }]
+        },
+        {
+          "comment": "text in italic",
+          "begin": "\\{i\\b",
+          "end": "\\}",
+          "contentName": "emphasis",
+          "patterns": [{ "include": "#markup" }]
+        },
+        {
+          "comment": "center text",
+          "begin": "\\{C\\b",
+          "end": "\\}",
+          "patterns": [{ "include": "#markup" }]
+        },
+        {
+          "comment": "left align text",
+          "begin": "\\{L\\b",
+          "end": "\\}",
+          "patterns": [{ "include": "#markup" }]
+        },
+        {
+          "comment": "right align text",
+          "begin": "\\{R\\b",
+          "end": "\\}",
+          "patterns": [{ "include": "#markup" }]
+        },
+        {
+          "comment": "unordered list",
+          "begin": "\\{ul\\b",
+          "end": "\\}",
+          "patterns": [{ "include": "#markup" }]
+        },
+        {
+          "comment": "ordered list",
+          "begin": "\\{ol\\b",
+          "end": "\\}",
+          "patterns": [{ "include": "#markup" }]
+        },
+        {
+          "comment": "link address text",
+          "begin": "\\{(?=\\{\\:)",
+          "end": "\\}",
+          "contentName": "string.other.odoc",
+          "patterns": [{
+            "begin": "\\{\\:",
+            "end": "\\}"
+        },
+        { "include":" #markup" }]
+        },
+        {
+        "comment": "string in source code",
+        "begin": "\\[",
+        "end": "\\]",
+        "contentName": "string.other.odoc",
+        "patterns": [{ "include": "#markup" }]
+      },
+      {
+        "comment": "string in preformatted source code",
+        "begin": "\\{\\[",
+        "end": "\\]\\}",
+        "contentName": "string.other.odoc",
+        "patterns": [{ "include": "#markup" }]
+      },
+      {
+        "comment": "string in verbatim",
+        "begin": "\\{v\\b",
+        "end": "v\\}",
+        "contentName": "string.other.odoc",
+        "patterns": [{ "include": "#markup" }]
+      },
+      {
+        "comment": "target-specific content",
+        "begin": "\\{\\%",
+        "end": "\\%\\}",
+        "contentName": "string.other.odoc",
+        "patterns": [{ "include": "#markup" }]
+      },
+      {
+        "comment": "cross-reference to an element",
+        "begin": "\\{\\!",
+        "end": "\\}",
+        "contentName": "string.other.odoc",
+        "patterns": [{ "include": "#markup" }]
+      },
+      {
+        "comment": "index table for given modules",
+        "begin": "\\{\\![a-z]+\\:",
+        "end": "\\]",
+        "contentName": "string.other.odoc",
+        "patterns": [{ "include": "#markup" }]
+      },
+      {
+        "comment": "insert a table of links",
+        "begin": "\\{\\!",
```

Missing the `indexlist` keyword.

```suggestion
        "begin": "\\{\\!indexlist",
```